packages/theme/src/components/menu.ts

```diff
@@ -56,9 +56,15 @@ export const Menu: ComponentMultiStyle<"Menu"> = {
         bg: ["blackAlpha.50", "whiteAlpha.50"],
       },
     },
+
```

Comment:
What is the intention of this blank line break?

Reply:
Unnecessary! I'm surprised you noticed! Thanks!